packages/dma-library/src/strategies/ajna/validation/borrowish/closeToMaxLtv.ts

```diff
@@ -7,7 +7,10 @@ export function validateGenerateCloseToMaxLtv(
   position: AjnaPosition,
   positionBefore: AjnaPosition,
 ): AjnaWarning[] {
-  if (position.maxRiskRatio.loanToValue.minus(MAX_LTV_OFFSET).lte(position.riskRatio.loanToValue)) {
+  if (
+    position.maxRiskRatio.loanToValue.minus(MAX_LTV_OFFSET).lte(position.riskRatio.loanToValue) &&
+    !positionBefore.debtAmount.eq(position.debtAmount)
```

Comment:
So it should be displayed only in a case when the debt changes? 

How about the case that withdrawing collateral will bring position to close to max ltv?

Reply:
for this case we have another validation below this one `validateWithdrawCloseToMaxLtv` :)

Reply:
Thanks! Approved!